ghost/admin/app/components/members/filter.hbs

```diff
@@ -84,7 +85,7 @@
                 </button>
                 <button
                     class="gh-btn gh-btn-primary"
-                    data-test-button="members-apply-filter" type="button" {{on "click" this.applyFilter}} {{on "keyup" this.handleSubmitKeyup}}
+                    data-test-button="members-apply-filter" type="button" {{on "click" (fn this.applyFiltersPressed dd)}} {{on "keyup" this.handleSubmitKeyup}}
```

Comment:
When you didn't make changes to the filter, the dropdown didn't close. That is why I created a new action here that also sends a close command to the dropdown.